### PR TITLE
fix(use-redirect-if-logged): check if organizations is fetched

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-905

+ And, remove useless redirect when we logout (causes a glitch)

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] staging - fix(use-redirect-if-logged): check if organizations is fetched - 2023-11-21T17:34:04Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/e8132fbc-9d5b-4fcb-86e0-c2923d505408/services/general)

 Another comment will be posted when deployments are terminated

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://977-console-staging.zc531a994.rustrocks.cloud)

File: libs/domains/organizations/feature/src/lib/hooks/use-organizations/use-organizations.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useQuery } from '@tanstack/react-query'
 import { queries } from '@qovery/state/util-queries'
 
-export function useOrganizations() {
+export function useOrganizations({ enabled }: { enabled?: boolean }) {
```
Comment (ctjhoa) [inline libs/domains/organizations/feature/src/lib/hooks/use-organizations/use-organizations.tsx:4]:
```suggestion
export function useOrganizations({ enabled }: { enabled?: boolean } = {}) {
```
It avoids specifying empty object every time

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/domains/organizations/feature/src/lib/hooks/use-organizations/use-organizations.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/domains/organizations/feature/src/lib/hooks/use-organizations/use-organizations.tsx]: https://github.com/Qovery/console/pull/977/commits/46b7b4ccdc2207ada2da4ef731d75d7e8569caae

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**